shell/platform/darwin/ios/framework/Source/FlutterTextInputPlugin.mm

```diff
@@ -387,8 +387,8 @@ - (void)dealloc {
 }
 
 - (UITextField*)textField {
-  if (_textField == nil) {
-    _textField = [[[UITextField alloc] init] autorelease];
+  if (!_textField) {
```

Comment:
Can you get rid of the `_textField` declaration above too please.  It is unnecessary because of the property declaration.

Reply:
You mean getting rid of https://github.com/flutter/engine/blob/5498add1982d73593cdc7766f7ccade43260e177/shell/platform/darwin/ios/framework/Source/FlutterTextInputPlugin.mm#L380-L382 ?

I thought this was needed because the property is `readonly` and I'm overriding its getter, so the ivar is not going to be automatically synthesized?